src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/.template.config/template.json

```diff
@@ -74,6 +74,82 @@
           "exclude": [
             "*.sln"
           ]
+        },
+        {
+          "condition": "(!IndividualLocalAuth || UseLocalDB)",
+          "exclude": [
+            "Server/app.db"
+          ]
+        },
+        {
+          "condition": "(!IndividualLocalAuth)",
+          "exclude": [
+            "Server/Data/SqlLite/**",
```

Comment:
This is weird. At first I thought it was a typo, since SQLite is called SQLite (one "L", not two), but then I see we spell it `SqlLite` in all the other templates too.

So I guess I agree we should have the same typo that exists in all the other templates 🤷‍♂

Reply:
No, it's SQLite, and yes, it also annoys me quite a bit :(.